public/jira/rollup/rollup.ts

```diff
@@ -82,9 +82,9 @@ interface RollupGroupedHierarchyOptions<CustomFields, Data, Meta> {
  * @param {IssueOrRelease} issueOrRelease
  * @returns {issueOrRelease is DerivedIssue}
  */
-export function isDerivedIssue(
-  issueOrRelease: IssueOrRelease
-): issueOrRelease is DerivedIssue {
+export function isDerivedIssue<CustomFields>(
+  issueOrRelease: IssueOrRelease<CustomFields>
+): boolean {
```

Comment:
Changing this to `boolean` makes it not usable for type narrowing anymore, which is usually what `isX` methods are for.